mindsdb/integrations/handlers/autosklearn_handler/autosklearn_handler.py

```diff
@@ -0,0 +1,44 @@
+from typing import Optional, Dict
+
+import dill
+import pandas as pd
+import autosklearn.classification as automl
+from autosklearn.metrics import accuracy
+
+from mindsdb.integrations.libs.base import BaseMLEngine
+
+
+class AutoSklearnHandler(BaseMLEngine):
+    """
+    Integration with the Auto-Sklearn ML library.
+    """
+
+    name = 'autosklearn'
+
+    def create(self, target: str, df: Optional[pd.DataFrame] = None, args: Optional[dict] = None) -> None:
+        if args is None:
+            args = {}
+
+        args['target'] = target
+
+        automl_classifier = automl.AutoSklearnClassifier(
+            time_left_for_this_task=60,
+            per_run_time_limit=360,
+            n_jobs=-1,
+            metric=accuracy
+        )
+        automl_classifier.fit(df.drop(target, axis=1), df[target])
```

Comment:
Just to double check: I take it that `df.drop` does not do this in-place, correct? Otherwise this would not work.

Reply:
No, it does not. Only if the `inplace` argument is set to `True` would it be executed that way.